DnD-Montreal/session-tome
# Implement action to create dm rating upon entry store/update

## Description
Closes #206 

Adds an action that creates a DM rating when rating data is passed in EntryStoreRequest/ EntryUpdateRequest.

## How to test
Store:
1. Create an entry that includes a valid rating_data array within the EntryStoreRequest
2. Query for a Rating that matches the created entry's id
3. Check if decimal value of Rating's 'categories' field matches the binary value corresponding to the rating categories specified as true in request

Store:
1. Create an entry
2. Make an update to the entry that includes a valid rating_data array in the EntryUpdateRequest
2. Query for a Rating that matches the created entry's id
3. Check if decimal value of Rating's 'categories' field matches the binary value corresponding to the rating categories specified as true in request
